lib/functions/compilation/uboot.sh

```diff
@@ -203,6 +203,19 @@ function compile_uboot_target() {
 		return 0 # exit after this
 	fi
 
+	##########################################
+	# REAL COMPILATION SECTION STARTING HERE #
+	##########################################
+
+	# Collect make environment variables, similar to 'kernel-make.sh'
+	uboot_make_envs=(
+		"CFLAGS='${uboot_cflags}'"
```

Comment:
@ColorfulRhino I'm again very late to the game, sorry: you're reading `uboot_cflags` before it is defined...